kubernetes/helm_charts/networkconfig/templates/blob-serviceentry-offline.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
@SMY-ALTAMASH Please rename the file to desktop-blob-serviceentry.yaml

Reply:
yup, looks better